# based on the assumption that the front drive actuator is the same as rear drive actuator, including transmission
front_left_wheel_autodriver:
  publish_rate: 50
  wheel_seperation_w: 0.1
  wheel_radius: 0.0765
  steer_joint_name: "front_steer_joint"
  rear_drive_joint_name: "rear_drive_joint"
